### PR TITLE
Topup balance

### PR DESCRIPTION
**Pull request by letier3110**
https://www.notion.so/madfissolutions/Top-up-balance-crypto-card-a70a967c5ff2482983542600b876fbc9

### CODE REVIEW

File: src/navigator/main-stack.tsx [hunk 1]
```diff
@@ -161,10 +161,17 @@ export const MainStackScreen = () => {
               />
 
               {/** Swap stack **/}
-              <MainStack.Screen
+              {/* <MainStack.Screen
                 name={ScreensEnum.Swap}
                 component={emptyComponent}
                 options={{ animationEnabled: false }}
+              /> */}
```
Comment (unixvb) [inline src/navigator/main-stack.tsx:168]: why this is commented?

File: src/screens/buy/buy.styles.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { createUseStyles } from '../../styles/create-use-styles';
+import { formatSize } from '../../styles/format-size';
+
+export const useBuyStyles = createUseStyles(({ colors, typography }) => ({
+  upperContainer: {
+    position: 'relative'
```
Comment (unixvb) [inline src/screens/buy/buy.styles.ts:6]: why?

File: src/screens/buy/disclaimer/disclaimer.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { Divider } from '../../../components/divider/divider';
+import { Icon } from '../../../components/icon/icon';
+import { IconNameEnum } from '../../../components/icon/icon-name.enum';
+import { formatSize } from '../../../styles/format-size';
+import { useDisclaimerStyles } from './disclaimer.styles';
+
+export const Disclaimer = () => {
```
Comment (unixvb) [inline src/screens/buy/disclaimer/disclaimer.tsx:10]: could you reuse the existing component? the same component already exist when you do reveal secrets

File: src/screens/buy/buy.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        <TextSegmentControl selectedIndex={tab.id} values={TABS.map(x => x.name)} onChange={handleTabChange} />
+        {tab.component}
+      </View>
+      {!tab.component && (
+        <DataPlaceholder
+          text={`Soon!
+We are working on that.`}
+        />
+      )}
+      {!isTablet() && <Disclaimer />}
```
Comment (unixvb) [inline src/screens/buy/buy.tsx:42]: why only for mobile?

**Review by unixvb — APPROVED** [review]

File: src/screens/buy/buy.tsx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  const publicKeyHash = useSelectedAccountSelector().publicKeyHash;
+
+  const selectedThemeIndex = theme === ThemesEnum.light ? 0 : 1;
+
+  const handleThemeSegmentControlChange = (newThemeIndex: number) =>
+    dispatch(changeTheme(newThemeIndex === 0 ? ThemesEnum.light : ThemesEnum.dark));
+
+  return (
+    <>
+      {/* <SettingsHeader /> */}
```
Comment (andrewmarushko) [inline src/screens/buy/buy.tsx]: Are we need this here?

File: src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-form-content.tsx [hunk 6]
```diff
@@ -49,7 +49,10 @@ export const RevealSeedPhraseFormContent: FC<FormikProps<RevealSeedPhraseModalFo
       />
       <RevealSeedPhraseView publicKeyHash={values.account.publicKeyHash} />
       <Divider size={formatSize(16)} />
-      <RevealAttention />
+      <Disclaimer
+        title="Attention!"
+        texts={['DO NOT share this set of chars with anyone!', 'It can be used to steal your current account.']}
```
Comment (andrewmarushko) [inline src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-form-content.tsx:54]: Maybe better use constants for static text somewhere in one place and use it everywhere?

Comment (letier3110) [reply src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-form-content.tsx:54]: there are different texts in different places of app

File: src/screens/buy/buy.tsx (hunk 5 above)
Comment (letier3110) [reply src/screens/buy/buy.tsx]: its slightly outdated commit commented, already fixed

**Review by andrewmarushko — APPROVED** [review]

**Pull request merged**